### PR TITLE
Login and SignUp frontend part

### PR DESCRIPTION
**Pull request by SeifMostafaa**
- Added the last frontend part which Gunjan worked on

### CODE REVIEW

File: frontend/src/app/classes/user.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+export class User {
+    public id: number
+    public name: string
+    public emailId: string
+    public username: string
+    public password: string
```
Comment (gunjanmimo) [inline frontend/src/app/classes/user.ts:6]: add one more field for user type, as in backend there I have added user type filed to distinguish user

File: frontend/src/app/signin/signin.component.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  <input
+    type="email"
+    name="email"
+    required
+    [(ngModel)]="user.emailId"
+    class="form-control"
+    placeholder="email"
+  />
+  <br />
+
```
Comment (gunjanmimo) [inline frontend/src/app/signin/signin.component.html:24]: may you add an option to choose user type?

File: frontend/src/app/classes/user.ts (hunk 1 above)
Comment (SeifMostafaa) [reply frontend/src/app/classes/user.ts:6]: I will, It's only the last version of the project and I will start working on it

**Review by gunjanmimo — APPROVED** [review]
LGTM

**Comment by tudorcodrea** [comment]
This will break the new Angular version we have. Therefore I am closing without merge

**Pull request closed without merging**